scripts/import-va-certs.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
No need for this to change.

Reply:
There is, or it breaks as the Dockerfile.datadog fails if this isn't changed. 
In the Dockerfile for api, it is being called with `bash` anyway, so this updated is appropriate and doesn't break the `docker-compose up` locally, nor broke it on deploy to dev.